packages/webapp/src/components/Finances/FinancesCarrousel/styles.module.scss

```diff
@@ -0,0 +1,219 @@
+  }
+}
+
+.expenseContainer {
+  padding: 4px;
+}
+
+.revenueTitle, .labourExpensesTitle, .otherExpensesTitle, .expenseTitle, .profitLossTitle {
+  font-family: 'Open Sans', 'SansSerif', serif;
+  font-size: 14px;
+  line-height: 18px;
+}
+
+.revenueTitle, .expenseTitle, .revenueStat, .expenseStat {
+  text-shadow: 0px 0.5px 0px #11615D;
+}
+
+.profitLossTitle, .profitLossStat {
+  text-shadow: 0px 0.5px 0px #A7CFC9;
+}
+
+.revenueContainer, .revenueTitle, .revenueStat {
+  color: #54FF8E;
+}
+
+.expenseTitle, .expenseStat {
+  color: #FFD900;
+}
+
+.profitLossTitle, .profitLossStat {
+  color: #3EA397;
+}
+
+.stat {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  font-family: 'Open Sans', 'SansSerif', serif;
+}
+
+.totalExpensesContainer {
+  background-color: #FFFAF0;
+  flex-grow: 1;
+  padding: 4px;
+  text-align: left;
+  border-radius: 4px 0 0 4px;
+}
+
+.expensesStatsContainer {
+  background-color: #FFF4D9;
+  flex-grow: 1;
+  padding: 12px 8px;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-radius: 0 4px 4px 0;
+
+  @media (min-width: 768px) {
+    padding: 24px 16px;
+  }
+}
+
+.totalExpensesTitle {
+  font-size: 12px;
+  line-height: 16px;
+  margin-bottom: 4px;
+}
+
+.expensesChartContainer {
+  position: relative;
+  height: 100px;
+  width: 100px;
+  margin: auto;
+
+  @media (min-width: 768px) {
+    height: 116px;
+    width: 116px;
+  }
+}
+
+.expensesChart {
+  position: absolute;
+  height: 100px;
+  width: 100px;
+
+  @media (min-width: 768px) {
+    height: 116px;
+    width: 116px;
+  }
+}
+
+.totalExpensesStatContainer {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.totalExpensesStat {
+  color: var(--red700);
+  text-shadow: 0px 1px 0px #FFC1BF;
+}
+
+.labourExpensesContainer, .otherExpensesContainer, .estimatedRevenueContainer, .revenueContainer {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  border-radius: 4px;
+  padding: 4px;
+  text-align: left;
+}
+
+.labourExpensesContainer, .otherExpensesContainer {
+  &:hover {
+    background-color: #FFEFCF;
+  }
+}
+
+.labourExpensesContainer {
+  color: var(--yellow700);
+}
+
+.otherExpensesContainer {
+  color: var(--brown700);
+}
+
+.labourExpensesTitle, .labourExpensesStat {
+  color: var(--yellow700);
+  text-shadow: 0px 0.5px 0px var(--brown700);
+}
+
+.otherExpensesTitle, .otherExpensesStat {
+  color: var(--brown700);
+  text-shadow: 0px 0.5px 0px var(--yellow700);
+}
+
+.estimatedRevenueCardContent {
+  background-color: var(--green400);
+  padding: 20px;
+  align-items: center;
+}
+
+.estimatedRevenueContainer, .estimatedRevenueTitle {
+  color: var(--teal700);
+}
+
+.estimatedRevenueTitle {
+  font-size: 12px;
+  font-family: 'Open Sans', 'SansSerif', serif;
+  line-height: 24px;
+}
+
+.estimatedRevenueContainer {
+  margin-left: 16px;
+
```

Comment:
I think I might prefer `min-width: max-content` on this one so that it doesn't break to two lines, but that might need revisiting after the other translations are supplied 🤔 

Otherwise maybe something with line-height? Or does that seem okay to you on mobile?

![Screenshot_20231031-121341](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/e61db4a6-12a1-4e0a-904a-c2a777b0795b)

Reply:
Yeah you're not being too fussy, this definitely looks weird to me too! Let me revisit